WootricSDK/WootricSDK/Wootric.h

```diff
@@ -77,4 +78,9 @@
 
 + (void)setCustomValueForResurveyThrottle:(NSNumber *)resurveyThrottle visitorPercentage:(NSNumber *)visitorPercentage registeredPercentage:(NSNumber *)registeredPercentage dailyResponseCap:(NSNumber *)dailyResponseCap;
 
++ (void)setSendButtonBackgroundColor:(UIColor *)color;
++ (void)setNoThanksColor:(UIColor *)color;
++ (void)setSliderColor:(UIColor *)color;
++ (void)setSocialSharingColor:(UIColor *)color;
```

Comment:
@diablourbano here I do need them, because these are the methods exposed to the user. And those properties are not declared in this class.